filip/clients/ngsi_v2/cb.py

```diff
@@ -157,6 +160,22 @@ def get_version(self) -> Dict:
             self.logger.error(err)
             raise
 
+    def _check_correct_cb_version(self) -> None:
+        """
+        Checks whether Orion version is >= 3.6.0, since breaking change was introduced there which introduced new
+        option, which is implemented in FiLiP, but won't work with lower orion versions.
+        """
+        orion_version = self.get_version()['orion']['version']
+        if version.parse(orion_version) < version.parse('3.6.0'):
```

Comment:
I think some significant changes have also been made in `3.8.0`. @tstorek Do you think we should also set an upper limit for the supported Orion version? like version < 3.8.0

Reply:
@djs0109 @jkriwet I think that it is worth having a look at the changes: https://github.com/telefonicaid/fiware-orion/releases. If introducing such bounds would also mean creating the issues to support the features in coming versions.